packages/@lwc/synthetic-shadow/src/faux-shadow/shadow-root.ts

```diff
@@ -140,6 +156,8 @@ export function attachShadow(elm: Element, options: ShadowRootInit): SyntheticSh
     setShadowRootResolver(sr, shadowResolver);
     // correcting the proto chain
     setPrototypeOf(sr, SyntheticShadowRoot.prototype);
+    // patch the host instance to support slotting
+    adoptHostToSupportSlotting(elm);
```

Comment:
if an element gets a shadow... it needs to be new methods for child interactions...